javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,48 @@
+var $words = document.querySelector('#sentence');
+var $submit = document.querySelector('.submit');
+var $middle = document.querySelector('.middle');
+var $HTML = document.querySelector('html');
+var $newEl = document.querySelectorAll('h1');
+
+var i = 0;
+
+function getSentence(event) {
+  event.preventDefault();
+  var words = $words.value;
+
+  for (var j = 0; j < words.length; j++) {
+    if (words[j] !== ' ') {
+      var newEl = document.createElement('h1');
+      newEl.append(words[j]);
+      $middle.append(newEl);
+    } else {
+      newEl = document.createElement('h1');
+      newEl.innerHTML = '&nbsp';
+      $middle.append(newEl);
```

Comment:
Rather than make an individual `h1` for each element, you might want to make a single `h1` that you add a child `span` element to for each letter in the phrase. If you do it that way, you may also be able to avoid using `&nbsp;`